content/en/docs/ops/configuration/multi-revision/gateways/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Configuring and managing gateways with multiple revisions
+description: Configuring and managing gateways with multiple revisions.
+weight: 25
+keywords: [kubernetes,upgrading]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+With a single `IstioOperator` CR, any gateways defined in the CR (including the `istio-ingressgateway` installed in the
+default profile) are upgraded in place, even when the canary control plane method is used. This is
+undesirable - gateways are a critical component that should be upgraded last, when the new version control and data
+planes are verified to be working.
+The recommended way to upgrade gateways is to define and manage them in a separate `IstioOperator` CR, separate from
+the one used to install and manage the control plane.
+
+{{< warning >}}
+To avoid problems with `.` (dot) not being a valid character in some Kubernetes paths, the revision name should not
+include `.` (dots).
+{{< /warning >}}
+
+## Istioctl
+
+This section covers installation and upgrade of separate control plane and gateways, using `istioctl`.
+
+### Installation with `istioctl`
+
+1.  Ensure that the main `IstioOperator` CR has a name and does not install a gateway:
+
+{{< text yaml >}}
+# filename: control-plane.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: `IstioOperator`
+metadata:
+  name: control-plane # REQUIRED
+spec:
+  profile: default
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        # MUST BE DISABLED
+        enabled: false
+{{< /text >}}
+
+1.  Create a separate `IstioOperator` CR for the gateway(s), ensuring that it has a name and has the `empty` profile:
+
+{{< text yaml >}}
+# filename: gateways.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: `IstioOperator`
+metadata:
+  name: gateways # REQUIRED
+spec:
+  profile: empty # REQUIRED
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        enabled: true
+{{< /text >}}
+
+Installation can be done with the following commands:
+
+{{< text bash >}}
+$ istioctl install -f control-plane.yaml --revision 1-8-0
+$ istioctl install -f gateways.yaml --revision 1-8-0
+{{< /text >}}
+
+Istioctl install and the operator track resource ownership through labels for both the revision and owning CR name.
+Only resources whose name and revision labels match the `IstioOperator` CR passed to `istioctl` install/operator will be
+affected by any changes to the CR - all other resources in the cluster will be ignored.
+It is important to make sure that each `IstioOperator` installs components that do not overlap with another `IstioOperator`
+CR, otherwise the two CR's will cause controllers or `istioctl` commands to fight.
+
+### Upgrade with `istioctl`
+
+Let's assume that the target version is 1.8.1.
```

Comment:
nit: I would use either the magic `{{<istio_version>}}` (not sure if that is right) or a fake version, otherwise this will just look stale in a month or 2?

Reply:
Good idea @howardjohn 
```suggestion
Let's assume that the target version is `{{< istio_version >}}.1`.
```

Reply:
On second thought, this would require a lot of rework in the rest of the section that is all based on 1.8.x as the example, so maybe it's best to just stick with the 1.8.1 point-in-time example.

Reply:
yes, i don't think the version can be automated because the examples have things like 1-8-1 based on the version.